### ISSUE TITLE
Sending reset command from uploader blocks a telemetry reconnect

### ISSUE THREAD

**Issue by peabody124** [TauLabs/TauLabs#1529]
If you go to uploader and connect reset, the flight controller resets correctly but the telemetry connection cannot be reestablished. This was using remote telemetry so no bootloader could be detected.

**Comment by joergrohde** [comment]
Sparky connects again after using connect button.

**Comment by peabody124** [comment]
Yeah, I believe this requires using radio telemetry (e.g. TauLink).

**Comment by joergrohde** [comment]
Mine has been connected directly with my NB and it happens. I use reset when I configure my FC so I don´t have to pull out the USB after every step.
